### ISSUE TITLE
Error executing 'postInstallation': Failed to connect to discourse:3000 after 36 tries

### ISSUE THREAD

**Issue by rene-gomez** [bitnami/bitnami-docker-discourse#50]
Hi.

In the firts time, when ejecuting "docker-compose up" for fisrt time, the sidekiq contaider down, because try to connect to discourse.
```


^[[36msidekiq_1     |^[[0m ^[[0m^[[1mWelcome to the Bitnami discourse container^[[0m
^[[36msidekiq_1     |^[[0m ^[[0mSubscribe to project updates by watching ^[[1mhttps://github.com/bitnami/bitnami-docker-discourse^[[0m
^[[32mpostgresql_1  |^[[0m nami    INFO  Initializing postgresql
^[[36msidekiq_1     |^[[0m ^[[0mSubmit issues and feature requests at ^[[1mhttps://github.com/bitnami/bitnami-docker-discourse/issues^[[0m
^[[32mpostgresql_1  |^[[0m postgre INFO  ==> No injected configuration files found. Creating default config file...
^[[36msidekiq_1     |^[[0m ^[[0m
^[[32mpostgresql_1  |^[[0m postgre INFO  ==> Deploying PostgreSQL from scratch...
^[[36msidekiq_1     |^[[0m nami    INFO  Initializing discourse-sidekiq
^[[36msidekiq_1     |^[[0m Error executing 'postInstallation': Failed to connect to discourse:3000 after 36 tries
^[[36mforum_sidekiq_1 exited with code 1

```

If I want up the container again the log says:
```

^[[36msidekiq_1     |^[[0m nami    INFO  discourse-sidekiq successfully initialized
^[[36msidekiq_1     |^[[0m Starting discourse-sidekiq...
^[[36msidekiq_1     |^[[0m nami    ERROR ELOOP: too many symbolic links encountered, lstat '/opt/bitnami/discourse-sidekiq/logs/sidekiq.log'
```

So I need destroy all containers (docker-compose down) and up all again (docker-compose up -d), after this moment the containers work!

Best regards

**Comment by beltran-rubo** [comment]
Sidekiq needs that discourse container is up & running at "discourse:3000", then the sidekiq process is started. Could you check if the discourse container is running and listening at 3000 port?

**Comment by rene-gomez** [comment]
those errors only appear in the first "docker-compose up"
best regards.

**Comment by beltran-rubo** [comment]
the first time you deploy Discourse takes more time to configure the whole application and to populate the database. In any case we think we added a big timeout for having enough time for fully deploy the application. We tested it in several scenarios.

Thanks for reporting it! We will check if we should increase more that timeout.

**Comment by rene-gomez** [comment]
Hi @beltran-rubo 
The problem is fixed in an especific version?

Best regards

**Comment by bhardin** [comment]
I also have this problem. However, if I kill it, and then restart it, I get:

```
postgresql_1  | ERROR:  extension "hstore" already exists
postgresql_1  | STATEMENT:  CREATE EXTENSION hstore;
discourse_1   | Error executing 'postInstallation': ERROR:  extension "hstore" already exists
discourse_1   |
discourse_discourse_1 exited with code 1
```

**Comment by rene-gomez** [comment]
That's correct @bhardin but the @beltran-rubo  say: everyting is ok!

@beltran-rubo  can you help us with this problem?, is necessary increase the timeout.

And please label "issue" for this.

best regards.

**Comment by beltran-rubo** [comment]
We released 2.1.1-debian-9-r16 that increases the timeout significantly for the Sidekiq server. @rene-gomez @bhardin please post if that fixes the issue on your side.

**Comment by rene-gomez** [comment]
yes @beltran-rubo works fine!

thanks for your support!

i close this issue.

best regards
